community/inventory_service/item-schema-automated.json

```diff
@@ -691,10 +727,13 @@
 			"itemdefid": 28,
 			"type": "item",
 			"name_english": "Priority Number One",
+			"name_italian": "Priorità Assoluta",
 			"name_japanese": "最優先事項",
 			"briefing_name_english": "Priority Number One",
+			"briefing_name_italian": "Priorità Assoluta",
```

Comment:
"Number One" is slang for peeing, just in case it's possible to get that pun in Italian.